feathr_project/feathr/definition/materialization_settings.py

```diff
@@ -31,6 +31,9 @@ def __init__(self, name: str, sinks: List[Sink], feature_names: List[str], backf
         self.name = name
         now = datetime.now()
         self.backfill_time = backfill_time if backfill_time else BackfillTime(start=now, end=now, step=timedelta(days=1))
+        for sink in sinks:
+            if isinstance(sink, RedisSink):
+                sink.aggregation_features = feature_names
```

Comment:
Should we do the same for HDFS sink as well?

Reply:
Will cause error when applying it for HDFS. Will open a new bug task for HDFS sink.